### DISCUSSION TITLE
Need your thoughts on Raycast extension for shiori

### DISCUSSION THREAD

**Discussion by namiwang** [go-shiori/shiori#839] [Q&A]
So I've created a simple extension for Raycast to add bookmarks to shiori in one click.

👉[extension](https://www.raycast.com/namiwang/shiori).
👉[source](https://github.com/raycast/extensions/tree/main/extensions/shiori)

![init](https://github.com/go-shiori/shiori/assets/1160688/e305b677-2746-46fc-ba01-4d3098ea3098)

I plan to add more features and if you use raycast, please let me know your feedbacks, thanks :)

**Comment by Monirzadeh** [comment]
Hi
i am not using Raycast personally but beside general functionality some feature like #826 can be interesting.
it good when you think extension is mature enough add that to the documentation too :+1: 

>Auto Tagging Rules
https://github.com/go-shiori/shiori/discussions/460
Automatically fill in fields on create
https://github.com/go-shiori/shiori/discussions/538

**Comment by fmartingr** [comment]
This looks amazing @namiwang! My only request on the current status is if we could just type `shiori URL` and press enter to add it to shiori, but honestly it's just a nit. Thank you for making this!

Would you be open to share this in our docs so users know this exists?

**Reply by namiwang** [reply]
Glad to help :), and sure please add it to the docs!

I don't know if it's possible with raycast to _just type `shiori URL`_ but I'll look into it.

**Reply by fmartingr** [reply]
I also don't know, not very familiar with it's SDK. Alternatively I can just create a keyboard shortcut for it, so it's not that huge of a deal, really.
